COMMITTERS.md

```diff
@@ -56,20 +56,19 @@ Our definition of Lead, Committers and Contributors roughly follows https://gith
 
 | Name | GitHub ID | Email | Roles |
 |-|-|-|-|
-| Billie Jean Simmons | @JillieBeanSim | BillieJean.Simmons@ibm.com | Lead, TSC Rep, Developer |
+| Billie Jean Simmons | @JillieBeanSim | BillieJean.Simmons@ibm.com | Co-Lead, Developer |
 | Fernando Rijo Cedeno | @zFernand0 | fernando.rijocedeno@broadcom.com | Co-Lead, Developer |
 | Jessica Tonda | @JTonda | jessica.tonda@broadcom.com | Scrum Master |
-| Ana Ceron | @anaxceron | ana.ceron@broadcom.com | Tech Writer |
+| Ana X. Ceron | @anaxceron | ana.ceron@broadcom.com | Tech Writer |
 | Rudy Flores | @rudyflores | rudyflores@ibm.com | Developer |
 | Trae Yelovich | @traeok | trae.yelovich@broadcom.com | Developer |
-| Timothy Johnson | @t1m0thyj | timothy.johnson@broadcom.com | Developer |
-| Adam Wolfe | @adam-wolfe | adam.wolfe@broadcom.com | Product Owner, Developer |
+| Timothy Johnson | @t1m0thyj | timothy.johnson@broadcom.com | TSC Rep, Developer |
+| Adam Wolfe | @adam-wolfe | adam.wolfe@broadcom.com | Security Squad Rep, Developer |
```

Comment:
Is @zFernand0 the Security Squad rep for Zowe Explorer?

Reply:
Fixed -- Fernando is sec rep for ZE; you are sec rep for CLI